src/core/components/table/CoachList.vue

```diff
@@ -185,6 +193,31 @@ export default {
       if (this.canSetUserCompany) userPayload.company = this.company._id;
       return userPayload;
     },
+    async nextStep () {
+      try {
+        this.loading = true;
+        const userInfo = await Users.exists({ email: this.newUser.local.email });
+        const user = userInfo.user;
+
+        const sameOrNoCompany = !user.company || user.company === this.company._id;
+        if (userInfo.exists && (get(userInfo, 'user.role.client') || !sameOrNoCompany)) {
+          NotifyNegative('Utilisateur déjà existant');
+        } else if (userInfo.exists) {
+          await Users.updateById(userInfo.user._id, { role: this.newUser.role, company: this.company._id });
+          NotifyPositive('Formateur créé');
+          await this.getUsers();
+          this.userCreationModal = false;
+          this.resetUserCreationForm();
```

Comment:
tu peux le mettre dans le @hide de la modal normalement

Reply:
du coup tu peux ne pas le mettre la car c'est fait automatiquement quand la modal se cache

Reply:
Oui mais c'est cette fonction qui permet de cacher la modale non ?

Reply:
ah je viens de voir : 
- ici il faut mettre : this.userCreationModal = false
- il ne faut pas le mettre dans le reset